Backends/PostScript/clim-postscript.asd

```diff
@@ -1,17 +1,17 @@
-
 ;;; CLIM-PostScript is not a backend in the normal sense.
 ;;; It is an extension (Chap. 35.1 of the spec) and is an
 ;;; "included" part of McCLIM. Hence the defsystem is here.
-;;; 
+;;;
 ;;; Move the whole system to the Modules/ directory - dk
 
 (defsystem #:clim-postscript
-  :depends-on (#:clim-basic #:clim-postscript-font)
+  :depends-on (#:clim-basic
+               #:mcclim-backend-common
+               #:clim-postscript-font)
```

Comment:
The new dependency is on the command output destination stuff in `Core/clim-core/commands.lisp`. How would you prefer to express this? `:depends-on (#:clim-core #:clim-postscript-font)`?

Reply:
I think it still depends also on clim-basic, so that would be

`:depends-on (#:clim-basic #:clim-core #:clim-postscript-font)`

otoh I don't really like the fact that postscript backend depends on core. I'd prefer to have all backends depend only on clim-basic (so they may be used without loading clim-core); main motive is to allow future speration for the low level abstraction (silica/dragonstone) and clim.

I understand that this is about output-destination in commands, but is there a chance to have some generic mechanism on top of which command mechanism is built?

Reply:
We could move the code for `invoke-with-standard-output`, `{stream,file}-destination`, `register-output-destination` and `*output-destination-types*` to the `mcclim-backend-common` system, i.e. a new file `Backends/common/output-destination.lisp`.